python/tskit/tables.py

```diff
@@ -1505,6 +1505,21 @@ def __banner(self, title):
         title_line += "#"
         return line + "\n" + title_line + "\n" + line + "\n"
 
+    def __iter__(self):
+        """
+        Iterate over the tables in this TableCollection, ordered by table name
+        (i.e. deterministically), returning a tuple of (table_name, table_object)
+        """
+        # For speed we simply list these by hand. They are tested against an
```

Comment:
We don't need this comment - it's perfectly obvious what the method is doing.

Reply:
Removed with https://github.com/tskit-dev/tskit/commit/04359b2d5bdba10c493746ba153505b3334a086f